regolith/helpers/u_milestonehelper.py

```diff
@@ -70,7 +71,14 @@ def db_updater(self):
         filterid = {'_id': key}
         target_prum = rc.client.find_one(rc.database, rc.coll, filterid)
         if not target_prum:
-            raise RuntimeError(f"Cannot find {key} in the projecta collection")
+            pra = fragment_retrieval(self.gtx["projecta"], ["_id"], rc.projectum_id)
+            if len(pra) == 0:
+                raise RuntimeError("Please input a valid projectum id or a valid fragment of a projectum id")
+            print("Projectum not found. Projectum with similar names: ")
+            for i in range(len(pra)):
+                print(f"{pra[i].get('_id')}")
+            print("Please rerun the helper with the correct name of the projectum.")
```

Comment:
"Please rerun the helper specifying the complete ID."

Reply:
done!